### ISSUE TITLE
Have you reproduced the results shown in the paper?

### ISSUE THREAD

**Issue by Jian-danai** [creotiv/hdrnet-pytorch#30]
Hi, have you reproduced the results on the FiveK dataset?

**Comment by Ssakura-go** [comment]
> Hi, have you reproduced the results on the FiveK dataset?

The size shown in the ds (input and output) are not match, but the results seems great. 
Have you get what you want by this code?
I'd like to get the results and compare it with my own method. If you are convenient, I am very glad to make a discussion with you. (#^.^#)

**Comment by Jian-danai** [comment]
> 

The PSNR on fivek is not stable even after 20~30 epochs training, typically +-0.3dB, maybe the lr is too large. Do you have any idea on it? Thanks.

**Comment by Sissuire** [comment]
> 

What is the batch size and learning rate ? I cannot reproduce the result and the gap is large. I'm seeking for the reason.

**Comment by Jian-danai** [comment]
> > 
> 
> What is the batch size and learning rate ? I cannot reproduce the result and the gap is large. I'm seeking for the reason.

just try default? batch_size=6

**Comment by Sissuire** [comment]
@Jian-danai Currently I set lr=1e-4, batch_size=24, and directly resize the input into 256x256/512x512 without any augmentation. And PSNR on fivek is 21.96dB. I don;t know whether it is good or bad :(

**Comment by Sissuire** [comment]
Further, I have also encountered the discontinuity problem. Even with global features, the result seems not good as the difference among grids is visible.

**Comment by Ssakura-go** [comment]
> Further, I have also encountered the discontinuity problem. Even with global features, the result seems not good as the difference among grids is visible.

Maybe we can talk on WeChat or other app where we are convenient to discuss.

**Comment by Ssakura-go** [comment]
> Maybe we can talk on WeChat or other app where we are convenient to discuss.

Maybe we can talk on WeChat or other app where we are convenient to discuss.
